packages/cache/src/salt.ts

```diff
@@ -25,7 +23,16 @@ function envHashKey(environmentGlobFiles: string[]) {
   return environmentGlobFiles.sort().join("|");
 }
 
-async function getEnvHash(environmentGlobFiles: string[], repoRoot: string) {
+function sortObject(unordered: Record<string, unknown>) {
```

Comment:
Is this needed? I was trying to follow usage but I only see it used in TargetHasher, which then pushes object values into an array which gets sorted within hashStrings. But maybe i'm missing a step.